src/commands/junit/api.ts

```diff
@@ -36,26 +36,30 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
     fileName = 'default_file_name'
   }
 
-  const spanTags: Record<string, string | undefined> = {
+  const metadata: Record<string, any> = {
     service: jUnitXML.service,
     ...jUnitXML.spanTags,
     '_dd.cireport_version': '2',
     '_dd.hostname': jUnitXML.hostname,
   }
 
   if (jUnitXML.logsEnabled) {
-    spanTags['_dd.junitxml_logs'] = 'true'
+    metadata['_dd.junitxml_logs'] = true
```

Comment:
This is on purpose.

While implementing this I changed how we decode in the backend so any JSON type is allowed now